### DISCUSSION TITLE
1.能否屏蔽自带的快捷键2.能否有一个只在播放界面生效的夜间模式（关灯模式）

### DISCUSSION THREAD

**Discussion by vtnP5fR7SxAaMyU** [the1812/Bilibili-Evolved#2382] [功能建议 / Ideas]
像W点赞，W投币的快捷键和自己的其他扩展程序快捷键冲突。
b站自带的关灯模式下连标题，导航栏，播放列表都没了,想要有个好用的关灯模式

**Comment by Geeyun-JY3** [comment]
如果开了 `快捷键扩展` 就能覆盖掉 B 站自带快捷键了。而且是能自定义的：https://github.com/the1812/Bilibili-Evolved/blob/preview/src/video/keymap/key-bindings.md
如果嫌上述方法麻烦或控制台看不懂，v2 有 GUI 可直接自定义。

**Comment by the1812** [comment]
> b站自带的关灯模式下连标题，导航栏，播放列表都没了,想要有个好用的关灯模式

关灯模式不就是隐藏这些东西用的吗, 不想隐藏直接开夜间模式吧

**Comment by Geeyun-JY3** [comment]
> 只在播放界面生效的夜间模式

这是不是可以通过 Stylus 安装 Bilibili-Evolved 夜间模式对应的那个用户样式然后调一调应用于的链接实现？

**Reply by the1812** [reply]
更新会覆盖掉吧...

**Reply by Geeyun-JY3** [reply]
关掉脚本的夜间模式然后开用户样式不行吗……

**Reply by the1812** [reply]
用户样式也是会自动更新的, 除非你把那里面的 updateURL 也删了

**Reply by Geeyun-JY3** [reply]
忘了这个了……
